doc/whats_new.rst

```diff
@@ -46,7 +46,7 @@ Requirements
 Bug fixes
 ^^^^^^^^^
 
-- ...
+- Fix m/f for male/female and l/r for left/right instead of only accpeting caps, by `Alex Rockhill`_ (:gh:`755`)
```

Comment:
you have a typo there :) also, try to phrase these entries such that they make sense for somebody who will never look at your PR. For example:

```suggestion
- :func:`mne_bids.make_report` now (1) detects male/female sex and left/right handedness irrespective of letter case, (2) will parse a ``gender`` column if no ``sex`` column is found in ``participants.tsv``, and (3) reports sex as male/female instead of man/woman, by `Alex Rockhill`_ (:gh:`755`)
```

Reply:
Thanks :) My bad I modeled it off of Alex Gramfort's last bug comment haha